pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -194,11 +227,13 @@ def __init__(
             self,
             wires,
             *,
+            mpi: bool = False,
+            mpi_buf_size: int = 0,
```

Comment:
Can we move these to below `batch_obs`?

Reply:
yes, we can. But I would prefer to keep it as it is since current interface is aligned with that of `lightning-gpu`.  Users would need extra work to update their scripts and we have to update all our blog posts and our posts in the forum.

Reply:
That's true only if the names aren't used for the variables, which we use everywhere explicitly. Let's make the move and lock it in.